Admin/Factory/FieldFactory.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace LAG\AdminBundle\Admin\Factory;
+
+use LAG\AdminBundle\Admin\Configuration\ApplicationConfiguration;
+use LAG\AdminBundle\Admin\Field;
+use LAG\AdminBundle\Admin\FieldInterface;
+use BlueBear\BaseBundle\Behavior\ContainerTrait;
+use Exception;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Field factory. Instances fields with its renderer.
+ */
+class FieldFactory
+{
+    use ContainerTrait;
+
+    protected $configuration;
+
+    protected $fieldsMapping = [];
+
+    public function __construct(ApplicationConfiguration $configuration)
```

Comment:
The configuration doesn't seem to be used atm, should it be used by the fields factory anyway?